flytekit/deck/deck.py

```diff
@@ -174,3 +174,29 @@ def get_deck_template() -> "Template":
         autoescape=select_autoescape(enabled_extensions=("html",)),
     )
     return env.get_template("template.html")
+
+
+class PythonDependencyDeck(Deck):
+    """
+    PythonDependencyDeck is a deck that contains information about the library name and version.
+    """
+
+    def __init__(self, name: Optional[str] = "Python Dependency", html: Optional[str] = ""):
```

Comment:
Nit: Usually there is more than one dependency

```suggestion
    def __init__(self, name: Optional[str] = "Python Dependencies", html: Optional[str] = ""):
```

Reply:
updated